feature-libs/product/future-stock/components/future-stock-accordion/future-stock-accordion.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { ICON_TYPE } from '@spartacus/storefront';
+import { FutureStock } from '../../root/models/future-stock.model';
+
+@Component({
+  selector: 'cx-future-stock-accordion',
+  templateUrl: './future-stock-accordion.component.html',
+  host: {
+    '[class.collapsed]': '!expanded'
+  }
+})
+export class FutureStockAccordionComponent {
+
+  @Input() header: string;
+  @Input() expanded: boolean = false;
+  @Input() content: FutureStock[] | string;
```

Comment:
I'm wondering about translations, in case there is no information about the future stock but the customer wants to display info in a specific language.

Worth considering is translation key handling as well.